spec/models/condition_spec.rb

```diff
@@ -1,4 +1,24 @@
 describe Condition do
+  context ".seed" do
+    it "should contain conditions" do
+      Condition.seed
+      specifications = YAML.load_file(File.join(ApplicationRecord::FIXTURE_DIR, "#{Condition.table_name}.yml"))
+      specifications.reverse!
+
+      expect(Condition.count).to eq(1)
+      Condition.all.each do |condition|
+        spec = specifications.pop
```

Comment:
Even though there's only 1 here, if this were changed to more than one, then this test would be non-deterministic because of the unknown database ordering.  In the case of conditions, the name is the unique bit, so you could probably change this to something like the following

```ruby
specifications = YAML.load_file(File.join(ApplicationRecord::FIXTURE_DIR, "#{Condition.table_name}.yml"))
specifications_by_name = specifications.index_by { |h| h[:name] }

expect(Condition.count).to eq(1)
Condition.all.each do |condition|
  spec = specifications_by_name[condition.name]
  ...
```

You could also do a sort by name on both sides, but then you have to deal with case and collation and in the end it's probably better to just to the Hash indexing.